LICENSE

```diff
@@ -290,6 +290,7 @@
    Trino
    ./integration-test-common/src/test/java/org/apache/gravitino/integration/test/util/CloseableGroup.java
    ./trino-connector/trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/hive/SortingColumn.java
+   ./trino-connector/integration-test/src/test/resources/trino-ci-testset/testsets/hive/00012_format.sql
 
```

Comment:
Why we need to add this line?

Reply:
Because these cases I refer to the cases in trino repo.